code/game/objects/items/miscellaneous.dm

```diff
@@ -277,3 +294,12 @@
 			var/atom/A = V
 			ouija_spaghetti_list[initial(A.name)] = A
 	return ouija_spaghetti_list
+
+/obj/item/upgradewand
+	desc = "A wand laced with nanotech calibration devices."
+	name = "Upgrade Wand"
+	icon = 'icons/obj/guns/magic.dmi'
+	icon_state = "nothingwand"
+	item_state = "wand"
+	w_class = WEIGHT_CLASS_SMALL
+	var/used = FALSE
```

Comment:
This variable is never actually used, only set. Once upgraded, it gets set to true, but there is no check that requires this to be false to upgrade something